File: Testing/Code for Testing/Basic/JS/M008_enums_constants_test.js
const Priority = Object.freeze({low: 1, medium: 2, high: 3});
const Color = Object.freeze({red: 10, green: 20, blue: 30});

const VERSION = 1;
const APP = "TestApp";
function main() {
    console.log(VERSION);
    console.log(APP);
    let p = Priority.high;
    console.log(p);
    let c = Color.green;
    console.log(c);
    if ((p === 3)) {
        console.log("high priority confirmed");
    }
    return 0;
}

function __ul_sort(arr, n) { const sub = arr.slice(0, n).sort((a, b) => a < b ? -1 : a > b ? 1 : 0); for (let i = 0; i < n; i++) arr[i] = sub[i]; }

main();

File: Testing/Code for Testing/Basic/JS/M008_enums_constants_test.py
from dataclasses import dataclass, field

def __ul_sort(arr, n): arr[:n] = sorted(arr[:n])

class __ul_Exception(Exception):
    def __init__(self, type_name, message):
        self.type_name = type_name
        self.message = message
        super().__init__(message)

from enum import IntEnum

class Priority(IntEnum):
    low = 1
    medium = 2
    high = 3

class Color(IntEnum):
    red = 10
    green = 20
    blue = 30

VERSION = 1
APP = 'TestApp'

def main():
    print(VERSION)
    print(APP)
    p = Priority.high
    print(p)
    c = Color.green
    print(c)
    if (p == 3):
        print('high priority confirmed')
    return 0


if __name__ == "__main__":
    main()
